### ISSUE TITLE
[DocDB] flaky test: org.yb.pgsql.TestPgColumnReadEfficiency.testScans

### ISSUE THREAD

**Issue by bmatican** [yugabyte/yugabyte-db#17822]
Jira Link: [DB-6910](https://yugabyte.atlassian.net/browse/DB-6910)
### Description

https://detective-gcp.dev.yugabyte.com/stability/test?analyze_trends=true&branch=master&build_type=all&class=org.yb.pgsql.TestPgColumnReadEfficiency&fail_tag=all&name=testScans&platform=linux

seems to trace to enabling packed for ysql

### Warning: Please confirm that this issue does not contain any sensitive information

- [X] I confirm this issue does not contain any sensitive information.

[DB-6910]: https://yugabyte.atlassian.net/browse/DB-6910?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by Huqicheng** [comment]
```
ts1|pid29824|:24294 Memory usage: 
ts1|pid29824|:24294 root: 1022.22M
ts1|pid29824|:24294  TCMalloc Central Cache Freelist: 4.22M
ts1|pid29824|:24294  TCMalloc PageHeap Freelist (Mapped): 117.62M
ts1|pid29824|:24294  TCMalloc PageHeap Freelist (Unmapped): 24.77M
ts1|pid29824|:24294  TCMalloc Sharded Transfer Cache Freelist: 0B
ts1|pid29824|:24294  TCMalloc Sum of CPU Cache Freelists: 3.72M
ts1|pid29824|:24294  TCMalloc Transfer Cache Freelist: 11.48M
ts1|pid29824|:24294  server: 104.99M (159.94M)
ts1|pid29824|:24294   BlockBasedTable: 66.85M
ts1|pid29824|:24294    IntentsDB-{t_tablet_id1}: 0B
ts1|pid29824|:24294    IntentsDB-{t_tablet_id2}: 16.01M
ts1|pid29824|:24294    IntentsDB-{t_v1_v2_v3_t1_t2_t3_t4_t5_idx_tablet_id1}: 50.79M
ts1|pid29824|:24294    IntentsDB-{t_tablet_id3}: 0B
ts1|pid29824|:24294    RegularDB-{t_tablet_id1}: 0B
ts1|pid29824|:24294    RegularDB-{t_tablet_id2}: 22.1K
ts1|pid29824|:24294    RegularDB-{t_v1_v2_v3_t1_t2_t3_t4_t5_idx_tablet_id1}: 19.5K
ts1|pid29824|:24294    RegularDB-{t_tablet_id3}: 0B
ts1|pid29824|:24294   Call: 2.8K
ts1|pid29824|:24294    Inbound RPC: 2.8K
ts1|pid29824|:24294    Outbound RPC: 0B
ts1|pid29824|:24294   Compressed Read Buffer: 0B (27.47M)
ts1|pid29824|:24294    Receive: 27.47M
ts1|pid29824|:24294   Read Buffer: 0B (27.47M)
ts1|pid29824|:24294    Inbound RPC: 0B (25.31M)
ts1|pid29824|:24294     Reading: 0B
ts1|pid29824|:24294     Receive: 25.31M
ts1|pid29824|:24294     Sending: 0B
ts1|pid29824|:24294    Outbound RPC: 0B (2.17M)
ts1|pid29824|:24294     Queueing: 0B
ts1|pid29824|:24294     Reading: 0B
ts1|pid29824|:24294     Receive: 2.17M
ts1|pid29824|:24294     Sending: 0B
ts1|pid29824|:24294   Tablets: 38.14M
ts1|pid29824|:24294    tablet-{t_tablet_id1}: 4.0K
ts1|pid29824|:24294     IntentsDB: 2.0K
ts1|pid29824|:24294      MemTable: 2.0K
ts1|pid29824|:24294     Metadata cache: 0B
ts1|pid29824|:24294     OperationsFromDisk: 0B
ts1|pid29824|:24294     RegularDB: 2.0K
ts1|pid29824|:24294      MemTable: 2.0K
ts1|pid29824|:24294     operation_tracker: 0B
ts1|pid29824|:24294    tablet-{transactions_tablet_id1}: 0B
ts1|pid29824|:24294     OperationsFromDisk: 0B
ts1|pid29824|:24294     operation_tracker: 0B
ts1|pid29824|:24294    tablet-{t_tablet_id2}: 5.88M
ts1|pid29824|:24294     IntentsDB: 5.88M
ts1|pid29824|:24294      MemTable: 5.88M
ts1|pid29824|:24294     Metadata cache: 0B
ts1|pid29824|:24294     OperationsFromDisk: 0B
ts1|pid29824|:24294     RegularDB: 2.0K
ts1|pid29824|:24294      MemTable: 2.0K
ts1|pid29824|:24294     operation_tracker: 0B
ts1|pid29824|:24294    tablet-{t_v1_v2_v3_t1_t2_t3_t4_t5_idx_tablet_id1}: 32.25M
ts1|pid29824|:24294     IntentsDB: 32.25M
ts1|pid29824|:24294      MemTable: 32.25M
ts1|pid29824|:24294     OperationsFromDisk: 0B
ts1|pid29824|:24294     RegularDB: 2.0K
ts1|pid29824|:24294      MemTable: 2.0K
ts1|pid29824|:24294     operation_tracker: 0B
ts1|pid29824|:24294    tablet-{transactions_tablet_id2}: 0B
ts1|pid29824|:24294     OperationsFromDisk: 0B
ts1|pid29824|:24294     operation_tracker: 0B
ts1|pid29824|:24294    tablet-{transactions_tablet_id3}: 0B
ts1|pid29824|:24294     OperationsFromDisk: 0B
ts1|pid29824|:24294     operation_tracker: 0B
ts1|pid29824|:24294    tablet-{t_tablet_id3}: 4.0K
ts1|pid29824|:24294     IntentsDB: 2.0K
ts1|pid29824|:24294      MemTable: 2.0K
ts1|pid29824|:24294     Metadata cache: 0B
ts1|pid29824|:24294     OperationsFromDisk: 0B
ts1|pid29824|:24294     RegularDB: 2.0K
ts1|pid29824|:24294      MemTable: 2.0K
ts1|pid29824|:24294     operation_tracker: 0B
ts1|pid29824|:24294    transactions: 0B
ts1|pid29824|:24294     transactions-{t_tablet_id1}: 0B
ts1|pid29824|:24294     transactions-{t_tablet_id2}: 0B
ts1|pid29824|:24294     transactions-{t_v1_v2_v3_t1_t2_t3_t4_t5_idx_tablet_id1}: 0B
ts1|pid29824|:24294     transactions-{t_tablet_id3}: 0B
ts1|pid29824|:24294   log_cache: 0B
ts1|pid29824|:24294    log_cache-{t_tablet_id1}: 0B
ts1|pid29824|:24294    log_cache-{transactions_tablet_id1}: 0B
ts1|pid29824|:24294    log_cache-{t_tablet_id2}: 0B
ts1|pid29824|:24294    log_cache-{t_v1_v2_v3_t1_t2_t3_t4_t5_idx_tablet_id1}: 0B
ts1|pid29824|:24294    log_cache-{transactions_tablet_id2}: 0B
ts1|pid29824|:24294    log_cache-{transactions_tablet_id3}: 0B
ts1|pid29824|:24294    log_cache-{t_tablet_id3}: 0B
```

**Comment by Huqicheng** [comment]
Track the fix with #18832
